libzeth/circuits/joinsplit.tcc

```diff
@@ -51,18 +51,18 @@ private:
     // which aggregate the extra bits and public values (+1)
     std::array<
         libsnark::pb_variable_array<FieldT>,
-        NumInputs + NumOutputs + 1 + NumInputs + 1>
+        2 * NumInputs + NumOutputs + 1 + 1>
```

Comment:
FYI: The `NumInputs` were not factored in the expression of the indices to refer explicitly to the ordering of the primary inputs